.teamcity/src/main/kotlin/projects/CheckProject.kt

```diff
@@ -24,8 +25,21 @@ class CheckProject(
         // Avoid rebuilding same revision if it's already built on another branch (pre-tested commit)
         param("teamcity.vcsTrigger.runBuildOnSameRevisionInEveryBranch", "false")
         param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%;%ge-td-dogfooding.grdev.net.access.key%")
-        param("additional.gradle.parameters", "")
-        param("reverse.dep.*.additional.gradle.parameters", "")
+
+        text(
+            "additional.gradle.parameters",
+            "",
+            display = ParameterDisplay.PROMPT,
+            allowEmpty = true,
+            description = "The extra gradle parameters you want to pass to this build, e.g. `--no-build-cache`"
```

Comment:
I think we should list the parameter `rerunAllTests` before `--no-build-cache`, since that is what you'd normally do.